packages/table/src/table.tsx

```diff
@@ -8,7 +8,6 @@
 import { AbstractComponent, Hotkey, Hotkeys, HotkeysTarget, IProps, Utils as CoreUtils } from "@blueprintjs/core";
 import * as classNames from "classnames";
 import * as React from "react";
-
```

Comment:
This newline was here intentionally. It's the boundary between "external" imports and "internal" imports. Can you add it back?

Reply:
ya, I will. my auto-fix removed it anyways. i assumed tied to prettier